pkgs/by-name/ya/yafc-ce/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDotnetModule,
+  dotnetCorePackages,
+  SDL2,
+  SDL2_image,
+  SDL2_ttf,
+}:
+buildDotnetModule rec {
+  pname = "yafc-ce";
+  version = "0.8.1";
+
+  src = fetchFromGitHub {
+    owner = "shpaass";
+    repo = "yafc-ce";
+    rev = version;
+    hash = "sha256-TihRIRD87a1bihIM+xj+bC6MSRT/8k8oiAGquuZpWBs=";
+  };
+
+  projectFile = "Yafc/Yafc.csproj";
+  nugetDeps = ./deps.nix;
+  dotnet-sdk = dotnetCorePackages.dotnet_8.sdk;
+  dotnet-runtime = dotnetCorePackages.dotnet_8.runtime;
```

```suggestion
  dotnet-sdk = dotnet.sdk;
  dotnet-runtime = dotnet.runtime;
```
*very* optional: use a `let...in` statement that binds `dotnet = dotnetCorePackages.dotnet_8`. (As seen [here](https://github.com/NixOS/nixpkgs/blob/686d942635db397457e2d77471cfca1d6948b9a4/pkgs/applications/audio/tagger/default.nix#L28-L29))
If you do use a `let...in` statement, you can remove the `rec` from the derivation and inherit attributes, like [this](https://github.com/NixOS/nixpkgs/blob/6002d45a9622accea71cdcbd1be17c465c5ae756/pkgs/by-name/sw/swayest-workstyle/package.nix#L7-L18).

---

Alternatively:
```suggestion
  dotnet-sdk = dotnetCorePackages.sdk_8_0;
  dotnet-runtime = dotnetCorePackages.runtime_8_0;
```
(As seen [here](https://github.com/NixOS/nixpkgs/blob/686d942635db397457e2d77471cfca1d6948b9a4/pkgs/development/tools/marksman/default.nix#L30-L31))

This is not a blocker and could be left as is.